catalog/app/components/Preview/load.js

```diff
@@ -10,6 +10,7 @@ import * as Json from './loaders/Json'
 import * as Markdown from './loaders/Markdown'
 import * as Ngl from './loaders/Ngl'
 import * as Notebook from './loaders/Notebook'
+import * as Package from './loaders/Package'
```

Comment:
`Manifest` would be cleaner imo, but whatever

Reply:
@fiskus feel free to just click resolve and merge

Reply:
I thought about that, and I don't mind renaming. But the directory name is `.quilt/packages` and this loader can be complemented by "NamedPackage" (just a link to manifest file): that's why I choose package, not manifest

Reply:
maybe, PackageManifest :thinking:

Reply:
or QuiltManifest, or QuiltPackageManifest =)
up to you

Reply:
I re-read https://docs.quiltdata.com/mentalmodel, and considered that "packages" directory contains packages each one containing list of manifest. So, `Manifest` name is more appropriate, I just unconsciously resist using it because it's too broad

I rename it in the next PR today

Reply:
Oops, it's the named_packages directory contains packages. But `.quilt/packages` contains list on manifest only.
Nevertheless, Manifest looks more appropriate